src/style.css

```diff
@@ -0,0 +1,84 @@
+@font-face {
+  font-family: "Avenir-book";
+  src: url(./fonts/Avenir-Book.ttf);
+}
+@font-face {
+  font-family: "Avenir-heavy";
+  src: url(./fonts/Avenir-Heavy.ttf);
+}
+
+* {
+  box-sizing: border-box;
+}
+
+.form-big,
+.form-small {
+  margin-top: 20px;
+  position: relative;
+}
+
+.big-search {
+  width: 100%;
+  height: 70px;
+  padding-left: 62px;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+
+  background: #fff;
+
+  font-family: "Avenir-book", sans-serif;
```

Comment:
add font-family for html tag, add here use `inherit`
https://developer.mozilla.org/en-US/docs/Web/CSS/inherit